### PR TITLE
Added / fixed tests and regex 

### PR DESCRIPTION
**Pull request by 0xmostafam**
Hello Again 👋

- Added tests and regex for Visual Studio Token and NPM Token
- Correct me if i am wrong but Bitly regex was incorrect because according to this video, https://www.youtube.com/watch?v=2rL4K4YOop8 and i even registered for a account to try it, the secret key was combosed of 40 hex characters
- I couldn't use `_assert_match_first_item` because Visual Studio Token and Bitly now has the same regex, and i noticed that there was an unused function called `_assert_match_in_items` so i rewrote it and used it for both of them

and if you want to me to refactor the tests so that all of them uses either `_assert_match_in_items` and `_assert_match_first_item` only i can do it

### CODE REVIEW

File: pywhat/Data/regex.json [hunk 1]
```diff
@@ -333,6 +333,19 @@
          "Bug Bounty"
       ]
    },
+   {
+      "Name": "NPM Token",
```
Comment (bee-san) [inline pywhat/Data/regex.json:337]: In your personal opinion, should we rename it to:

```suggestion
      "Name": "Node Package Manager (NPM) Token",
```

? Or is NPM so popular that people know it by acronym?

**Review by bee-san — COMMENTED** [review]
Some great work here 👏👏👏👏 Thank you!!! 💖💖💖

File: tests/test_regex_identifier.py [hunk 2]
```diff
@@ -24,7 +24,10 @@ def test_regex_successfully_parses():
 
 def _assert_match_in_items(name, res):
```
Comment (piatrashkakanstantinass) [inline tests/test_regex_identifier.py:25]: Can you change this to:
```py
def _assert_match_in_items(name, res):
    assert any(name in i["Regex Pattern"]["Name"] for i in res)
```
?

Comment (0xmostafam) [reply tests/test_regex_identifier.py:25]: sure

**Review by bee-san — APPROVED** [review]

**Pull request merged**